FWCore/Utilities/test/BuildFile.xml

```diff
@@ -36,7 +36,6 @@
 </bin>
 <bin    file="test_catch2_*.cc" name="testFWCoreUtilitiesCatch2">
   <use name="catch2"/>
-  <use name="FWCore/Utilities"/>
```

Comment:
@fabiocos Thanks. I just noticed this myself while preparing backports of #28561. Will this PR go in quickly-enough for a specific PR to fix it not being worth of the effort?

Reply:
@makortel @Dr15Jones well, it depends mostly on ```core``` decision :-)

Reply:
@makortel @Dr15Jones the dependency of PhysicsTools (and RecoHI) on GuiBrowsers was through https://github.com/cms-sw/cmssw/blob/master/FWCore/GuiBrowsers/python/ConfigToolBase.py that I moved to ```FWCore/Utilities``` Please let me know whether you see a more suitable place for it.
I found it a bit bizarre (I would imagine it is there for historical reasons)

Reply:
> the dependency of PhysicsTools (and RecoHI) on GuiBrowsers was through https://github.com/cms-sw/cmssw/blob/master/FWCore/GuiBrowsers/python/ConfigToolBase.py that I moved to `FWCore/Utilities` Please let me know whether you see a more suitable place for it.
> I found it a bit bizarre (I would imagine it is there for historical reasons)

The `ConfigToolBase.py` depends on `FWCore.ParameterSet`, and there is python code (`VarParsing`) in `FWCore.ParameterSet` that depends on `FWCore.Utilities`. It would be cleaner to place `ConfigToolBase.py` elsewhere to avoid circular dependencies of packages.

The comment string in `ConfigToolBase` says
https://github.com/cms-sw/cmssw/blob/44ebbe11472af7b4789e11a6fa8e4c130e43a321/FWCore/Utilities/python/ConfigToolBase.py#L35
which makes me think `PhysicsTools.PatAlgos` or `PhysicsTools.PatUtils` a natural place for it (on C++ level `PatAlgos` depends on `PatUtils`, but on python level there is code in `PatAlgos` depending on `PatUtils` and vice versa). This choice would mean `RecoHI.HiEgammaAlgos.HiCoreTools` to depend on `PhysicsTools.Pat*`, which actually is the case already. In addition, `HiCoreTools` (and `HiHelperTools`) seem to be unused, so one could think of trying to remove them as well.

Reply:
yes PhysicsTools.PatUtils looks more suitable place for it